HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 History
 =======
 
+3.13.1 (2020-06-04)
+-------------------
+
+Fix bugs relating to "after:" steps in data Load and record types in data Extract.
```

Comment:
```suggestion
Issues closed:

* Fixed a bug with "after:" steps in the `load_dataset` task.
* Fixed a bug with record types in the `extract_dataset` task.
```

(I try to refer to tasks in the release notes with the names that users are used to invoking)